.github/workflows/oci-conformance-action.yml

```diff
@@ -36,7 +36,7 @@ jobs:
           echo "SERVER_URL=http://${IP}:8080" >> $GITHUB_ENV
     - uses: actions/checkout@v3
       with:
-        # TODO: change to upstream once the foloowing PR is merged:
+        # TODO: change to upstream once the folowing PR is merged:
```

Comment:
Actually these comments should be removed, that PR was merged and we are already pointing to upstream.